lms/djangoapps/courseware/tests/test_word_cloud.py

```diff
@@ -223,10 +226,15 @@ def test_word_cloud_constructor(self):
             'ajax_url': self.block.ajax_url,
             'display_name': self.block.display_name,
             'instructions': self.block.instructions,
-            'element_class': self.block.location.block_type,
             'element_id': self.block.location.html_id(),
             'num_inputs': 5,  # default value
             'submitted': False,  # default value,
         }
 
-        assert fragment.content == self.runtime.render_template('word_cloud.html', expected_context)
+        if settings.USE_EXTRACTED_WORD_CLOUD_BLOCK:
+            expected_context['range_num_inputs'] = range(5)
```

Comment:
Why not check the block type in this case?

Reply:
We have updated the implementation.
Actually In Extracted Block we didn't define the `location` propoerty which exists in `xmodule` via `XModuleMixin` mixin [here](https://github.com/openedx/edx-platform/blob/master/xmodule/x_module.py#L329)
In linkage of it we have done reimplementation now [here](https://github.com/openedx/xblocks-contrib/pull/4/files#diff-a651b3990c862f9cca58a8c8d835edbe052ddbe28fd983eda11caede0bcce424R134) to fix the test case without change